cdap-security/src/main/java/io/cdap/cdap/security/authorization/AuthorizerWrapper.java

```diff
@@ -178,6 +178,16 @@ public void enforce(EntityId entity, Principal principal, Set<? extends Permissi
     }
   }
 
+  @Override
+  public void enforce(EntityType entityType, EntityId parentId, Principal principal, Permission permission)
+    throws AccessException {
+    try {
+      authorizer.enforce(parentId, principal, Action.ADMIN);
```

Comment:
Are we making the assumption this will always be admin? If that's the case, should we consider changing the name to enforceParent in order to make it clear that this is specifically for enforcing parented permissions?

Reply:
Well, this is fallback and for fallback we mostly mapping everything to admin. E.g. List Profiles in Namespace would map to Admin in Namespace in old model, that corresponds to original checks we had.

Reply:
As for method naming it's a debate between overloading or not :). I can remove overloading, it will probably be something like enforceOnParent, but it would be more verbose.

Reply:
Good point on the backwards compatibility thing, that should be fine. I suppose my primary concern is somebody who is implementing a new check accidentally using this method for a non parented check which would not work properly. If this isn't as big of a concern, we can leave it as-is :)

At the very least, we should properly document the function, and maybe consider changing the `entityType` variable name to `childEntityType` or something similar?

Reply:
Sure. renamed to prevent confusion

Reply:
Thanks!